conf/routes

```diff
@@ -216,7 +216,7 @@ GET   /frontend/channel-switches                       controllers.ChannelSwitch
 POST  /frontend/channel-switches/update                controllers.ChannelSwitchesController.set
 
 # ----- campaigns ----- #
-GET   /frontend/campaigns                              controllers.CampaignsController.get
+GET   /frontend/campaigns                              controllers.CampaignsController.get()
```

Comment:
This is for the warning 

[warn] /Users/xxxxxx/code/support-admin-console/conf/routes:219:1: Auto-application to `()` is deprecated. Supply the empty argument list `()` explicitly to invoke method get,
[warn] or remove the empty argument list from its definition (Java-defined methods are exempt).
[warn] In Scala 3, an unapplied method like this will be eta-expanded into a function. [quickfixable]
[warn] GET   /frontend/campaigns                              controllers.CampaignsController.get